app/com/lucidchart/open/cashy/views/search/index.scala.html

```diff
@@ -24,12 +26,25 @@
     @for(result <- results) {
       <p>
         <a href="@cashyRoutes.BrowseController.index(result.bucket, result.parent)" target="_blank"><span class="glyphicon glyphicon-folder-open"></span></a>
-        <a href="@result.link" target="_blank">@result.bucket/@result.key</a>
+
+        @if(query.isDefined && !query.get.contains("%")) {
```

Comment:
highlighting gets weird when they use their own wildcards, so we don't do it in that case